js/imageSlider.js

```diff
@@ -0,0 +1,110 @@
+const imageLinks = [
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628373/team-rebel/Home-01/Ellipse.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628407/team-rebel/Home-02/Ellipse_big.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628443/team-rebel/Home-03/Ellipse.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628208/team-rebel/About-01/Ellipse.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628579/team-rebel/Team-02/Team-4.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628407/team-rebel/Home-02/Ellipse_big.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628286/team-rebel/About-04/Ellipse.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628478/team-rebel/Service-01/Ellipse.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628580/team-rebel/Team-02/Team-3.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628550/team-rebel/Team-01/Team-3.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628549/team-rebel/Team-01/Team-1.png",
+    "https://res.cloudinary.com/john-pels/image/upload/v1568628548/team-rebel/Team-01/Team-5.png"
+];
+const h3Texts = [
+    'Lorem ipsum is placeholder text commonly  <br> text commonly',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, <br> text commonly',
+    'sed do eiusmod tempor incididunt ut labore et dolore  <br> text commonly',
+    'magna aliqua. Ut enim ad minim veniam, quis nostrud  <br> text commonly',
+    'exercitation ullamco laboris nisi ut aliquip ex ea  <br> text commonly',
+    'commodo consequat. Duis aute irure dolor in  <br> text commonly',
+    'reprehenderit in voluptate velit esse cillum dolore eu  <br> text commonly',
+    'fugiat nulla pariatur. Excepteur sint occaecat cupidatat  <br> text commonly',
+    'non proident, sunt in culpa qui officia deserunt mollit <br>  text commonly',
+    'Lorem ipsum, or lipsum as it is sometimes known, is dummy text  <br> text commonly',
+    'used in laying out print, graphic or web designs. The passage is <br>  text commonly',
+    'attributed to an unknown typesetter in the 15th century who is <br>  text commonly'
+]
+const tagline = [
+    'Kerry Johnes — Some Company',
+    'Seun  Agbeye — Andela ',
+    'Chidi Okoye — Node.js ',
+    'Olufemi O.  — Front-End Dev',
+    'Arafah O.  —  Front-End Dev',
+    'John O.  —  Front-End Dev',
+    'Olanrewaju O.  —  Front-End Dev',
+    'Zainab A.  —  Front-End Dev',
+    'Muhktar L.  —  Front-End Dev',
+    'Samuel N.  —  Front-End Dev',
+    'Idris A.  —  Front-End Dev',
+    'Tunde A.  —  Back-End Dev'
+]
+
```

Comment:
Nice, you can have an array of objects instead of this.
```
[{
 link: <image_link>,
description: <image_desc>,
role: <user_role>
}]
```

Reply:
Copy that.